src/tests/ftest/recovery/pool_list_consolidation.py

```diff
@@ -278,19 +279,20 @@ def test_lost_majority_ps_replicas(self):
         self.log_step("Remove <scm_mount>/<pool_uuid>/rdb-pool from two ranks.")
         scm_mount = self.server_managers[0].get_config_value("scm_mount")
         rdb_pool_path = f"{scm_mount}/{self.pool.uuid.lower()}/rdb-pool"
-        command = f"sudo rm {scm_mount}/{self.pool.uuid.lower()}/rdb-pool"
+        command = f"sudo rm {rdb_pool_path}"
         hosts = list(set(self.server_managers[0].ranks.values()))
         count = 0
         for host in hosts:
             node = NodeSet(host)
             check_out = check_file_exists(hosts=node, filename=rdb_pool_path, sudo=True)
             if check_out[0]:
-                pcmd(hosts=node, command=command)
+                run_remote(log=self.log, hosts=node, command=command)
```

Comment:
`run_remote` doesn't raise an exception, so we need to check for errors. E.g.
```suggestion
                if not run_remote(log=self.log, hosts=node, command=command).passed:
                    self.fail(f'Failed to remove {rdb_pool_path} on {host}')
```

Reply:
Done